packages/react-compose/package.json

```diff
@@ -27,7 +27,7 @@
     "tslib": "^1.9.3"
   },
   "peerDependencies": {
-    "react": "^16.3.2"
+    "react": "^16.8.1"
```

Comment:
Does it actually need such a high version constraint?

Reply:
Fair point. I thought we were intending to use hooks everywhere, but I see now that this is not one of the places on our hooks roadmap, so no need to limit consumers to react `16.8.x` 👍 1+